src/.gitignore

```diff
@@ -0,0 +1,2 @@
+check_configuration
+src/check_configuration
```

Comment:
Please move the content of this file into the "Compile Programs" section of wazuh/.gitignore and remove this file.